jerry-core/mem/mem-poolman.c

```diff
@@ -37,9 +37,9 @@
 /**
  * Node for free chunk list
  */
-typedef struct
+typedef struct mem_pools_chunk
```

Comment:
Thanks for fixing this. However, there are some more recursive `struct`s in the code base, which solve the the issue inconsitently, sometimes even incorrectly. Could you please open an issue or a follow-up PR to have a uniform solution for all recursive `struct`s out there?